version.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "2.14",
```

Comment:
I think we would need to bump the version as we did not follow the conventions.

Reply:
lol. as always, thanks for checking my work :)